src/styles/shared/_buttons.scss

```
@@ -288,7 +288,7 @@
 }
 
 @function plain-button-background() {
-  @return rgba(color('ink'), 0.1);
+  @return var(--p-surface-foreground-subdued, rgba(color('ink'), 0.1));
```

This is where my [alpha](https://github.com/Shopify/polaris-ux/issues/385) question came from.

---

That said, should this become the focus rectangle since these are all applied on focus to plain buttons?

---

A couple things here: First to make this match monochromatic links it would be:

```suggestion
  @return var(--p-neutral-action, rgba(color('ink'), 0.1));
```

Second this style should only get applied to the active state, and focus should only get the blue focus ring.

---

Let’s drop this change for now